src/Adapter/Product/PageConfiguration.php

```diff
@@ -53,6 +53,7 @@ public function getConfiguration()
         return [
             'display_quantities' => $this->configuration->getBoolean('PS_DISPLAY_QTIES'),
             'allow_add_variant_to_cart_from_listing' => $this->configuration->getBoolean('PS_ATTRIBUTE_CATEGORY_DISPLAY'),
+            'use_combination_image_in_listing' => $this->configuration->getBoolean('PS_USE_COMBINATION_IMAGE_IN_LISTING'),
```

Comment:
You should set the default value in the installer fixtures (install-dev/data/xml/configuration.xml)

Reply:
And add it during the upgrade

Reply:
@eternoendless It's there, check configuration.xml. :-)